src/test/java/utils/DriverFactory.java

```diff
@@ -0,0 +1,32 @@
+package utils;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.ie.InternetExplorerDriver;
+
+import java.io.File;
+
+public class DriverFactory {
+    private static WebDriver driver;
+    private static final String DRIVER_PATH = "src/test/resources/";
+
+    public static WebDriver getDriver(Browser browser) {
+        File file;
+        switch (browser) {
+            case CHROME:
+                file = new File(DRIVER_PATH + "chromedriver.exe");
```

Comment:
do you really need the 'file' variable here or you can use 'new File(DRIVER_PATH + "chromedriver.exe")' instead of it?

Reply:
I need this variable because I use it in the next step -  file.getAbsolutePath()